### PR TITLE
Feat : text component

### PR DESCRIPTION
**Pull request by alex-mngt**

### CODE REVIEW

File: src/components/Text/typescript/text.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  FontSizes,
+  FontWeights,
+} from '../../../shared/style/constants'
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Atoms/Text',
+  component: Text,
+  args: {
+    children: 'Greg made a wooden quantic cpu',
```
Comment (Damien-MATHIEU) [inline src/components/Text/typescript/text.stories.tsx:17]: Nice

File: src/components/Text/typescript/text.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import Text from '..'
+import {
+  Colors,
+  FontFamilies,
+  FontSizes,
+  FontWeights,
+} from '../../../shared/style/constants'
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
```
Comment (nartydev) [inline src/components/Text/typescript/text.stories.tsx:12]: We can remove comment from example

File: src/components/Text/typescript/text.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  args: {
+    children: 'Greg made a wooden quantic cpu',
+    font: FontFamilies.INTER,
+    size: FontSizes.BASE,
+    weight: FontWeights.MEDIUM,
+    color: Colors.WHITE,
+  },
+} as ComponentMeta<typeof Text>
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
```
Comment (nartydev) [inline src/components/Text/typescript/text.stories.tsx:25]: Same

File: src/components/Text/typescript/text.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+const Template: ComponentStory<typeof Text> = (args) => <Text {...args} />
+
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
```
Comment (nartydev) [inline src/components/Text/typescript/text.stories.tsx:28]: Same

**Comment by nartydev** [comment]
We can also remove examples of Storybook

**Review by Damien-MATHIEU — APPROVED** [review]

**Review by nartydev — APPROVED** [review]

**Review by Damien-MATHIEU — APPROVED** [review]

**Pull request merged**